stable/database/templates/statefulset.yaml

```diff
@@ -608,3 +608,185 @@ spec:
   {{- end }}
 {{- end }}
 {{- end }}
+
+{{- if .Values.database.te.enableStatefulSet }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
This is identical to the Deployment defined in `stable/database/templates/deployment.yaml` except for the changes I am calling out here.

1) Change Deployment to StatefulSet.